packages/vscode/syntaxes/yyp.tmLanguage.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Thanks for this! I'm still too new to TextMate to have been able to figure out this straight-forward of an approach to this problem.